lib/burnside/bridge.cc

```diff
@@ -52,8 +52,7 @@ L::cl::opt<bool> ClDisableToDoAssert("disable-burnside-todo",
     L::cl::Hidden);
 
 #undef TODO
-#define TODO() \
-  assert(false && "not implemented yet")
+#define TODO() assert(false && "not implemented yet")
```

Comment:
Do we have different versions of clang-format?

Reply:
I missed that change... It could be a version issue. I was using clang-format from llvm8 for FIR. Until there it seemed to give the same results. Which one are you using ?

Reply:
The one I build with the project. So, version 10.